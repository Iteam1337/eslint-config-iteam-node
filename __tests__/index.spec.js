const index = require('../')

it('create config', () => {
  expect(index).toMatchSnapshot()
})
