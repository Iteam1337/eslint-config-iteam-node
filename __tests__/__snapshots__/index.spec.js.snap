// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create config 1`] = `
Object {
  "env": Object {
    "jest": true,
    "mocha": true,
    "node": true,
  },
  "extends": Array [
    "eslint:recommended",
    "prettier",
  ],
  "parserOptions": Object {
    "ecmaVersion": "2018",
  },
  "rules": Object {
    "no-console": "off",
    "no-unexpected-multiline": "error",
    "no-var": "error",
    "prefer-const": "error",
    "semi": Array [
      "error",
      "never",
    ],
    "space-before-blocks": Array [
      "error",
      "always",
    ],
  },
}
`;
